examples/khr-3hv/full project/controllers/supervisorsManager/khr3hvEnv.py

```diff
@@ -0,0 +1,202 @@
+import numpy as np
+from scipy.ndimage.interpolation import shift
+
+from RobotUtils import RobotFunc
+from deepbots.supervisor.controllers.robot_supervisor import RobotSupervisor
+from gym.spaces import Box, Discrete
+
+import wandb 
+
+GLOBAL_DISTANCE = 0
+
+class khr3hvRobotSupervisor(RobotSupervisor):
+    
+    def __init__(self, projectName, wandbSaveFreq, train, useRay):
+        """
+        In the constructor the observation_space and action_space are set and references to the various components
+        of the robot required are initialized here. 
+        For the observation we are using a time window of 5, to store the last 5 episode observations.
+        Observation:
+            Type: Box(10)
+            Num	Observation             Min(rad)      Max(rad)
+            0   Robot position x-axis   -inf          +inf      
+            1   Robot position y-axis   -inf          +inf
+            2   Robot position z-axis   -inf          +inf
+            3   Robot velocity z-axis    0            +7  
+            4	LeftAnkle               -inf          +inf
+            5	LeftCrus                -inf          +inf
+            6	LeftFemur               -inf          +inf
+            7	RightAnkle              -inf          +inf
+            8	RightCrus               -inf          +inf
+            9	RightFemur              -inf          +inf
+             
+        Actions:
+            Type: Continuous
+            Num	BodyPost     Min      Max      Desc
+            0	LeftAnkle   -2.35    +2.35    Set the motor position from -2.35 to +2.35 
+            1	LeftCrus    -2.35    +2.35    Set the motor position from -2.35 to +2.35 
+            2	LeftFemur   -2.35    +2.35    Set the motor position from -2.35 to +2.35 
+            3	RightAnkle  -2.35    +2.35    Set the motor position from -2.35 to +2.35 
+            4	RightCrus   -2.35    +2.35    Set the motor position from -2.35 to +2.35 
+            5	RightFemur  -2.35    +2.35    Set the motor position from -2.35 to +2.35 
+          
+        Reward: 
+            bonus for moving forward 
+            difference of current and pre vious positions
+
+        Starting State:
+            [0, 0, ..., 0]
+        
+        Episode Termination:
+            Robot y axis smaller than 0.50 cm
+            Robot walked more that 15 m
+        """
+
+        super().__init__()
+        # Time window of 5
+        self.t = 5
+        # Number of observations
+        self.numObs = 10
+        # Total observations
+        self.obs = np.zeros(self.t * self.numObs)
+        # Lower and maximum values on observation space
+        lowObs = -np.inf * np.ones(self.t * self.numObs)
+        maxObs = np.inf * np.ones(self.t * self.numObs)
+        self.observation_space = Box(low=lowObs, high=maxObs, dtype=np.float64)
+        # Lower and maximum values on action space
+        lowAct = -2.356 * np.ones(6)
+        maxAct = 2.356 * np.ones(6)
```

Comment:
These two lines cause user warning.
_UserWarning: We recommend you to use a symmetric and normalized Box action space (range=[-1, 1])_
See [this link](https://stable-baselines.readthedocs.io/en/master/_modules/stable_baselines/common/env_checker.html).
```suggestion
        lowAct = -1 * np.ones(6)
        maxAct = 1 * np.ones(6)
```
Also, we may need to modify the related code in other sections to keep this action space, i.e., `apply_action`